app/src/views/private/components/export-sidebar-detail.vue

```diff
@@ -244,6 +247,19 @@ interface Props {
 	search?: string;
 }
 
+const userStore = useUserStore();
+const permissionsStore = usePermissionsStore();
+
+const createAllowed = computed(() => {
```

Comment:
Understandably this is based on other existing usage, but let's do this instead (other similar usage should probably get updated to this shorter syntax in the future):

```js
const createAllowed = computed<boolean>(() => permissionsStore.hasPermission(collection.value, 'create'));
```

Similar to:

https://github.com/directus/directus/blob/1ec4fe165b674c413efc0ee708f2425191e0f6d1/app/src/interfaces/select-dropdown-m2o/select-dropdown-m2o.vue#L201-L204

Also let's make sure to put this `createAllowed` at the bottom rather than before the `props` here.